Bottom: 96c82548efcc268b338f9c6ba44cd4566ef9a562
Top:    421dcb8b2218f3102f41a13ae3138203a2ead777
Author: Ian Kent <raven@themaw.net>
Date:   2013-06-07 13:00:32 +0800

Refresh of autofs-5.0.7-add-null-check-in-extract_version.patch

---

diff --git a/lib/mounts.c b/lib/mounts.c
index 0caa0aa..7b959b8 100644
--- a/lib/mounts.c
+++ b/lib/mounts.c
@@ -168,6 +168,8 @@ unsigned int get_kver_minor(void)
 static int extract_version(char *start, struct nfs_mount_vers *vers)
 {
 	char *s_ver = strchr(start, ' ');
+	if (!s_ver)
+		return 0;
 	while (*s_ver && !isdigit(*s_ver)) {
 		s_ver++;
 		if (!*s_ver)
